server/game/CardAbility.js

```diff
@@ -184,7 +184,7 @@ class CardAbility extends ThenAbility {
         let costMessages = this.cost.map(cost => {
             if(cost.getCostMessage) {
                 let card = context.costs[cost.getActionName(context)];
-                if(card && card.facedown) {
+                if(card && card.isFacedown && card.isFacedown()) {
                     card = 'a facedown card';
                 }
```

Comment:
other checks that call facedown don't check whether facedown exists but here you do for some reason?